app/src/controllers/appInfo/selectors.js

```diff
@@ -65,6 +66,8 @@ export const instanceTypeSelector = (state) =>
   environmentSelector(state)[INSTANCE_TYPE] || NOT_PROVIDED;
 export const allowDeleteAccountSelector = (state) =>
   environmentSelector(state)[ALLOW_DELETE_ACCOUNT] === 'true';
+export const userSuggestionsSelector = (state) =>
```

Comment:
Maybe rename it to match the boolean type?
F.e. `areUserSuggestionsAllowedSelector`.

Reply:
Done.